### PR TITLE
Droplimb Chance Tweak

### PR DESCRIPTION
**Pull request by cybercapitalism**
## What Does This PR Do

Fixes two oversights involving sharp weapon delimb chance.

How it works right now is that the chance of delimbing per attack with a sharp weapon is half of the damage that the attack deals to the limb itself. This is a problem because when you reach the limb's damage cap, half or more of your attack's damage is then distributed to other limbs. An esword attacking a head that is at its damage cap of 75 health will not have a 15% chance of delimbing, but rather a 7.5% chance or lower depending on what organs the damage is split between. 

The second oversight is that there was a 5% chance to deal internal organ damage when attacking with a brute weapon. This had the effect of halving the damage of the hit and dealing negligible damage to a random internal organ. It now deals half the damage of the hit to the internal organ and full external damage from the hit.
## Why It's Good For The Game

Delimbing heads for brain steal objectives is no longer a noob trap that involves amputation surgery and a circular saw or a changeling armblade. I've seen countless people smack a body for minutes without delimbing the head.
This will also effectively buff all sharp melee weapons, as they will now pose a much greater threat of lopping limbs off of people. 

## Testing

Verified that the limbs of skrells detached at a statistically acceptable rate. 
## Changelog
:cl:
tweak:doubles delimb chance to correct for oversight. you must now be in crit to be delimbed.
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
Please update this branch from master to resolve CI issues that were fixed in https://github.com/ParadiseSS13/Paradise/pull/21301

File: code/modules/surgery/organs/organ_external.dm [hunk 1]
```diff
@@ -269,9 +268,9 @@
 	//If limb took enough damage, try to cut or tear it off
 	if(owner)
 		if(sharp && !(limb_flags & CANNOT_DISMEMBER))
-			if(brute_dam >= max_damage && prob(brute / 2))
+			if(brute_dam >= max_damage && owner.health <= HEALTH_THRESHOLD_CRIT && prob(brute))
```
Comment (lewcc) [inline code/modules/surgery/organs/organ_external.dm:271]: This change isn't documented in the PR, and effectively means you can *only* delimb people with sharp weapons if they're in crit first. That's...kind of a big change, and if it was intentional, please update the PR body to reflect it.

**Review by lewcc — CHANGES_REQUESTED** [review]

**Comment by efzapa** [comment]
will eswords/chainsaws still delimb before crit as was their appeal and/or mechanic?

**Comment by cybercapitalism** [comment]
No, but you only need to be below 0 health to start delimbing. This is 4 hits for the e-sword and 2 hits for the chainsaw. If this change were not made, the incidence of instant decaps/delimbs would become very high and overbearingly powerful.

**Comment by matttheficus** [comment]
Not a fan of requiring crit for delimbs, at least for arms/legs/etc.

**Comment by Rurik123** [comment]
Requiring crit sounds fine for head delimb, but for arm/legs... ehhhhh. Not entirely sold.

**Review by S34NW — APPROVED** [review]

**Pull request merged**